### PR TITLE
Add Localization to Project

### PR DESCRIPTION
**Pull request by shrihari689**
## Proposed Changes

- [x] Add Basic Localization Setup
- [x] Add Language Selector
- [x] Make Some Initial Translations
- [x] Write Docs

📍 Docs: https://github.com/coronasafe/care_fe/blob/da1e18aa3a55a6663c53687a6cc178be3200b746/src/Locale/TRANSLATION_CONTRIBUTION.md

Minimal Translation Completed: 

- All Auth Pages and its forms
- SideBar
- Facilities Page

### CODE REVIEW

File: src/Locale/ta/Entities.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  "Facilities": "Facilities",
+  "Patients": "Patients",
+  "Sample Test": "Sample",
+  "Shifting": "Shifting",
+  "Resource": "Resource",
+  "External Results": "External",
+  "Users": "Users",
+  "Profile": "Profile",
+  "Dashboard": "Dashboard"
+}
```
Comment (bodhish) [inline src/Locale/ta/Entities.json]: @shrihari689 It would be great if you translate these 😄

**Review by bodhish — APPROVED** [review]
LGTM

**Comment by bodhish** [comment]
@shrihari689 can you fix the merge conflicts?

File: src/Locale/ta/Entities.json (hunk 1 above)
Comment (shrihari689) [reply src/Locale/ta/Entities.json]: Wohooooo 🥳 Done @bodhish 🚀

**Comment by shrihari689** [comment]
Done 🎉 @bodhish

**Pull request merged**